src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -105,6 +106,61 @@
             });
         }
 
+        #createTableSearchFields() {
+            document.querySelectorAll('div.table-with-search-field').forEach((table, i) => {
+                const searchField = document.createElement('input');
+                searchField.type = 'search';
+                searchField.placeholder = 'search...';
+                searchField.id = `table-search-field-${i}`;
+                searchField.classList.add(`table-search-field-input`);
+                searchField.autocapitalize = 'off';
+                searchField.autocomplete = 'off';
+                searchField.autocorrect = 'off';
+                table.insertBefore(searchField, table.firstChild);
+
+                const labelField = document.createElement('label');
+                labelField.htmlFor = `table-search-field-${i}`;
+                labelField.classList.add(`table-search-field-label`);
+                labelField.innerHTML = 'Search inside the contents of the table';
+                table.insertBefore(labelField, table.firstChild);
+
+                searchField.addEventListener('input', () => {
+                    const query = searchField.value.toLowerCase();
+                    table.querySelectorAll('tbody tr').forEach((row) => {
+                        let rowMatchesQuery = false;
+                        for (const cell of row.querySelectorAll('td, th')) {
```

Comment:
if you want to make this faster (not sure how big the tables can be in the profiler), you could use `for (const cell of row.cells)` instead of querying the DOM to find the table cells. As we don't have nested tables, this will give you the same result faster (and if we had nested tables, this might actually _fix_ things)